cmd/sebak/cmd/cmd_test.go

```diff
@@ -17,10 +18,56 @@ func TestParseFlagSelfValidators(t *testing.T) {
 	flagValidators = "https://localhost:12346?address=GDPQ2LBYP3RL3O675H2N5IEYM6PRJNUA5QFMKXIHGTKEB5KS5T3KHFA2"
 	flagKPSecretSeed = "SCN4NSV5SVHIZWUDJFT4Z5FFVHO3TFRTOIBQLHMNPAZJ37K5A2YFSCBM"
 	flagBindURL = "http://0.0.0.0:12345"
+
 	parseFlagsNode()
-	vs, err := parseFlagValidators("self")
-	require.Nil(t, err)
-	require.Equal(t, 1, len(vs))
-	validator := vs[0]
-	require.Equal(t, bindEndpoint.Host, validator.Endpoint().Host)
+	require.Equal(t, 2, len(localNode.GetValidators()))
+
+	parsedValidator, _ := node.NewValidatorFromURI(flagValidators)
+	validator := localNode.GetValidators()[parsedValidator.Address()]
+
+	require.Equal(t, validator.Address(), parsedValidator.Address())
+	require.Equal(t, validator.Endpoint().Host, parsedValidator.Endpoint().Host)
+	require.Equal(t, validator.Endpoint().Port(), parsedValidator.Endpoint().Port())
```

Comment:
How about adding test case only "self" in flagValidators?
Because there is no test case for specifying "self".